packages/react-components/src/link/src/useLink.ts

```diff
@@ -1,6 +1,32 @@
-import { cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
+import { ForwardedRef } from "react";
+import { MergedRef, Size, cssModule, mergeClasses, normalizeSize, useAutoFocus, useMergedRefs } from "../../shared";
 import { isNumber } from "lodash";
 
+export interface UseLinkProps {
+    cssModule?: string;
```

Comment:
can't know at the moment if they are required props here. Should i let them all optional except forwardRef?

Reply:
yeah leave everything optionnal and i'll update them sometime